package.json

```diff
@@ -28,12 +30,10 @@
 	"devDependencies": {
 		"@biomejs/biome": "^1.6.4",
 		"@octokit/types": "^12.6.0",
-		"@types/node": "^16.18.96",
+		"@types/bun": "^1.1.0",
 		"@types/semver": "^7.5.8",
-		"@vercel/ncc": "^0.38.1",
 		"lefthook": "^1.6.8",
-		"typescript": "^5.4.5",
-		"unbuild": "^2.0.0"
+		"typescript": "5.5.0-dev.20240417"
```

Comment:
Was this required to make the build pass? I'd prefer if we used a stable version.

Reply:
It was just to fix `tsc --noEmit`. Bun does not do type checking, the build works just fine.